litmus-portal/graphql-server/utils/misc.go

```diff
@@ -110,12 +134,21 @@ func ManifestParser(cluster dbSchemaCluster.Cluster, rootPath string, subscriber
 		newContent = strings.Replace(newContent, "#{LITMUS-CHAOS-EXPORTER}", subscriberConfig.ChaosExporterImage, -1)
 		newContent = strings.Replace(newContent, "#{ARGO-CONTAINER-RUNTIME-EXECUTOR}", subscriberConfig.ContainerRuntimeExecutor, -1)
 
+		if len(*cluster.NodeSelector) > 0 || cluster.NodeSelector != nil {
```

Comment:
Just checking if `nodeSelector!=""` should work too I guess
```suggestion
		if nodeSelector!="" {
```

Reply:
Yes, this part got left. I added and removed from other places

Reply:
nodeSelector!="" . This will not work. `cluster.NodeSelector != nil` is enough in this case

Reply:
Made the change